source/jobs/index.md

```diff
@@ -2,32 +2,32 @@
 layout: formatted-text
 title: Join the team
 ---
-<p class="intro-copy">There’ll soon be more seats around Convivio’s big table. It’s been a busy summer already and we’re planning to expand our team.</p>
+<p class="intro-copy">We are not currently hiring for any positions.</p>
 
-We prefer building relationships to the traditional ‘CV out of the blue’ approach — so we’d like to start hearing from people now who’d be interested in joining us in the coming months.
+We're always looking to hear from people who would be interested in working with us in the future. Drop us a line at <a href="mailto:jobs@weareconvivio.com">jobs@weareconvivio.com</a>
 
-If you’re looking for a new challenge in digital, then Convivio could be the perfect next move for your career.
+## What it's like to work at Convivio
 
-## What are we like?
+At Convivio we're not just remote, [we're free-range.](https://blog.weareconvivio.com/free-range-working-an-introduction-27eb178db97c) We don't have a central office and we all work from home. We encourage flexible working. [Read more about working from home at Convivio.](/jobs/working-from-home)
 
-We’re an enthusiastic, collaborative and friendly team who are quickly building a great agency to help our customers solve some big problems.
+We're a self-managing company, we all have a high-degreee of ownership over our work environment, process, and output.
 
-We focus on designing and building digital services — helping large public and private sector organisations move their core processes online.
+We're open and transparent, and believe in doing things right. We're the first digital agency to be [awarded the fair tax mark.](https://blog.weareconvivio.com/convivio-becomes-first-digital-agency-to-be-awarded-the-fair-tax-mark-881021825741)
 
-We work in very open ways, and often together with other partner agencies, or clients’ in-house teams.
+We offer unlimited holiday, an unlimited equipment budget, and unlimited training budget.
```

Comment:
Just raising this as a discussion point prior to publishing this page …

I know we have 'unlimited holiday' now in our practices, and our published policy relfects that (https://cookbook.weareconvivio.com/business-operation-recipe/taking-time-off-work/holiday) but I noticed with the updated contract stuff that Steve's been working on that they still have holidays down as a defined number of days. I wonder whether we need to have another review or discussion and maybe an updated draft of our holiday policy to clarify things exactly.

For instance, we might rephrase the Cookbook page to say we have XX days (what's in the contract) and we're continuing to experiment with unlimited holiday.

Anyway, just wanted to raise it as a discussion point at this stage.

Reply:
I think I remember having this discussion with Steve H early on and he said the contract is more restrictive than our official policy just as a safeguard against abuse. @steveparks might have input to give here as it might affect our legal obligations.

Reply:
Yeah, employment law, and therefore employment contracts, hasn't quite caught up with free range... so the compromise we've adopted is:
1. The contract sets out a 'standard' employment approach, which is the default
2. It also provides for 'Free range' as a flexitime-plus scheme that staff and the company can agree to enrol them in. The benefits of free range aren't defined in the contract, but in a separate policy. In case of problems, the working can be reverted to 'standard' as part of performance review or disciplinary procedures, making the contractual benefits and obligations clear.
3. We can then write up all of our extra freedoms under a 'free range' policy in the cookbook (to be worked on once we'd finished the update of the contract), and this can include the unlimited holiday, peer review and so on.
4. We'll offer everyone free range. It's a trust to be lost rather than earned

So the way to phrase this might be along the lines of: "All colleagues are invited to work Free Range - giving unlimited holiday, expenses, equipment budget and more freedoms, subject to peer review on a case by case basis."

Reply:
Could we make it more informal? It sound more like text you'd find in the contract than on the website jobs section?

Reply:
Or could it be a footnote?

Reply:
I've made some changes, mostly adding the line about peer reviews with a link to the cookbook.

Reply:
Yeah, your point about being more informal is a good one, and your changes on this work well. I'd perhaps edit to be even more casual, such as:
"We offer unlimited holiday, expenses, equipment and training budget, and more freedoms — with peer review rather than management authorisation."

Reply:
Great thanks Steve, that is a lot better. I've pushed that change